prow/config/config_test.go

```diff
@@ -1026,7 +1026,7 @@ func TestSecretAgentLoading(t *testing.T) {
 						tempSecret, changedTokenValue, tempSecretValue))
 				} else {
 					// Secret agent needs some time to update the values. So wait and retry.
-					time.Sleep(100 * time.Millisecond)
+					time.Sleep(400 * time.Millisecond)
```

Comment:
Add more retries but keep the sleep the same

Reply:
can we not do real sleeps in unit tests? :/

Reply:
I'd rather not have it either, but I'm not sure of a better way -- this test mucks with the filesystem and it might take some time for the notifier to trigger and update. It's async but I guess we could refactor it to allow us to throw a channel in and block on that @droslean

Reply:
hmm, fwiw filesystem notifications are not going to behave super portably, we've had problems with tests like this in k/k in the past... I had to make /tmp an actual tmpfs for the integration tests to work around this. It's probably best not to test the fs notification package.

Reply:
eek.